content/webapp/pages/collections/index.tsx

```diff
@@ -35,11 +35,7 @@ const Page: NextPage<
       staticContent={
         <ContaineredLayout gridSizes={gridSize12()}>
           <SpacingSection>
-            <SearchForm
-              searchCategory="works"
-              location="page"
-              showTypewriter={true}
-            />
+            <SearchForm searchCategory="works" location="page" />
```

Comment:
I've just realised we don't have a piece to add the checkbox, do you think it's better to open a new ticket for it? Should it live in the SearchForm, what do you think?
<img width="328" height="93" alt="Screenshot 2025-09-17 at 17 36 40" src="https://github.com/user-attachments/assets/9b1bf0dc-b8f1-4747-ac3b-485135914309" />
(not for this PR)

Reply:
Probably warrants a new ticket so it doesn't get lost, aye

Reply:
#12290